tests/test_faster_rcnn.py

```diff
@@ -24,6 +24,8 @@
 )
 from datasetinsights.io.checkpoint import EstimatorCheckpoint
 
```

Comment:
Whats the purpose of adding instructions here? Cant we create a task and fix it?